doc/plotting/index.rst

```diff
@@ -428,7 +428,7 @@ use :func:`view_markers`.
 :func:`view_img_on_surf`: Surface plot using a 3D statistical map::
 
     >>> from nilearn import plotting, datasets     # doctest: +SKIP
-    >>> img = datasets.fetch_localizer_button_task()['tmaps'][0]     # doctest: +SKIP
+    >>> img = datasets.fetch_localizer_button_task()['tmaps']     # doctest: +SKIP
```

Comment:
Its not longer 'tmaps' but 'tmap'. Push a commit with this change. Perhaps it may reappear.